q2_feature_table/_summarize/summarize_assets/sample-frequency-detail.html

```diff
@@ -1,35 +1,182 @@
-{% extends "tab-child.html" %}
+{% extends "tabbed.html" %} {% block head %}
+<script src="js/vega.min.js"></script>
+<script src="js/vega-embed.min.js"></script>
+<link rel="stylesheet" type="text/css" href="css/spinkit.css" /> {% endblock %}
+{% block tabcontent %}
 
-{% block head %}
-  <script src="./dist/vendor.bundle.js"></script>
-{% endblock %}
+<!-- template it in using jinja -->
```

Comment:
I'm not sure this comment adds anything to the discussion: you template the spec in using jinja on the next line. I like the approach of: code tells you _how_ something is done, comments tell you _why_ something was done. Food for thought.